Include/patchlevel.h

```diff
@@ -33,3 +35,7 @@
                         (PY_MICRO_VERSION <<  8) | \
                         (PY_RELEASE_LEVEL <<  4) | \
                         (PY_RELEASE_SERIAL << 0))
+
+#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= 0x030B0000
+PyAPI_DATA(const unsigned long) Py_Version;
```

Comment:
I didn't expect that adding Py_Version to patchlevel.h would require to change the main Python.h, and require to add a pyport.h to patchlevel.h. The C API is fragile, please don't move includes.

Since you set the variable in Python/getversion.c which implements Py_GetVersion(), and Py_GetVersion() is exposed in pylifecycle.h, please move Py_Version definition in pylifecycle.h and revert the include changes.